### PR TITLE
New page to view objects waititng for qPCR

### PR DESCRIPTION
**Pull request by aanil**
New page requested by Christian to view samples/pools waiting for qPCR from all flows.

### CODE REVIEW

**Comment by alneberg** [comment]
Is it possible to do something about this alignment of columns? 
Screenshot 2019-11-19 at 11 39 43

File: run_dir/design/qpcr_pools.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<div class="container-fluid" id="qpcrpools_list">
+  <div id="pools_table_div">
+      <h1><span id="page_title">qPCR pools</span>
+        <small>Showing all qPCR pools</small>
+      </h1>
+      <table class="table table-striped table-bordered sortable" id="pools_table">
+        <thead>
+          <tr class="sticky">
+            <th class="sort"></th>
+            <th class="sort">Pool</th>
```
Comment (ewels) [inline run_dir/design/qpcr_pools.html]:
```suggestion
            <th class="sort w-50">Pool</th>
```

Comment (ewels) [reply run_dir/design/qpcr_pools.html]: Wait, sorry - that's only with Bootstrap 4x 😞 (we should probably upgrade one day..)

```suggestion
            <th class="sort" style="width:50%">Pool</th>
```

Comment (aanil) [reply run_dir/design/qpcr_pools.html]: Made some changes, they should line up now!

**Comment by ewels** [comment]
I know this is already merged, but now that I'm on a computer and can actually see the page I have a few UI suggestions:

- [ ] `Waiting(in days)` > `Waiting (in days)` (add space)
- [ ] Make the whole table cell clickable, or at least the name. Not just the `+` icon
- [ ] Use [coloured labels](https://getbootstrap.com/docs/3.4/components/#labels) to give visual indication of how old the wait time is
- [ ] Use labels for the `(3)` sample count information _(I assumed that this was part of the WS name at first)_
- [ ] Add `table-hover` class to tables so that it's easier to follow the rows with the mouse cursor
- [ ] Have buttons at the top of the page to expand-all and collapse-all rows

These are all optional, so ignore if too much work. But most are trivial and should make the page significantly easier to read I think.

Note that I am being very restrained here - the use of tables and huge quantities of white space pains me greatly 😆 But I can see that this is a quick and functional addition so I'm not suggesting that you completely rewrite the page just to satisfy my OCD.

Nice work @aanil!

**Comment by aanil** [comment]
Added a trello card for the rest apart from the first!

**Pull request merged**